### ISSUE TITLE
Code error "This declaration shadows a built in symbol"

### ISSUE THREAD

**Issue by dharte93** [ethereum/mist#4046]
- [ ] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

```
Version: `0.0.0`
OS & Version: windows/linux/osx
Node version: `geth 0.0.0`
Number of blocks synchronized: 0
```

**Comment by evertonfraga** [comment]
Hi @dharte93, 

can you be more specific about this issue? Where did you see that error message, and under what circumstances?

please provide the full stack trace or error message. Thanks a bunch

**Comment by fredbuluku** [comment]
I can’t remember the issue maybe if reminded because I have dealt with
many,but no problem if there’s no one at place by now. Thanks

Regards,

Fred.

**Comment by evertonfraga** [comment]
I have reasons to believe this error was due to Solidity compiler version mismatch bug. Which is fixed on #4052. Please let me know otherwise.
